examples/multiple_tables/main.tf

```diff
@@ -22,15 +22,29 @@ provider "google" {
 }
 
 module "bigquery" {
```

Comment:
Instead of editing the existing example, can we add a specific "authorized view" example?